help_to_heat/portal/download_views.py

```diff
@@ -181,22 +183,16 @@ def download_feedback_view(request):
     return response
 
 
-@require_http_methods(["GET"])
-@decorators.requires_service_manager
-def download_referrals_all_view(request):
-    referrals = portal_models.Referral.objects.all().order_by("referral_id")
-    downloaded_at = timezone.now()
-    file_name = downloaded_at.strftime("%d-%m-%Y %H_%M")
-    response = create_referral_xlsx(referrals, file_name, exclude_pii=True)
-    return response
-
-
 @require_http_methods(["GET"])
 @decorators.requires_service_manager
 def download_referrals_last_week_view(request):
-    referrals = portal_models.Referral.objects.filter(created_at__gte=datetime.now() - timedelta(days=7)).order_by("referral_id")
-    downloaded_at = timezone.now()
-    file_name = downloaded_at.strftime("%d-%m-%Y %H_%M")
+    today = date.today()
```

Comment:
Please can you add a comment to explain the boundaries, like:
```
# Weekly boundaries are Mondays at midnight (00:00).
today = …
end_of_last_week = …
start_of_last_week = …
```

It's not immediately obvious from the code, so good to help out future devs.

Reply:
Added!